features/step_definitions/work_download_steps.rb

```diff
@@ -11,7 +11,7 @@
 end
 
 Then /^I should receive a file of type "(.*?)"$/ do |filetype|
-  mime_type = filetype == "azw3" ? "application/x-mobi8-ebook" : MIME::Types.type_for("foo.#{filetype}").first
+  mime_type = filetype == "azw3" ? "application/x-mobi8-ebook" : Marcel::MimeType.for(name: "foo.#{filetype}").to_s
```

Comment:
Unless I'm missing some CI shenanigans (i.e. I've only tried running the tests locally), we don't seem to be needing the exception on azw3 anymore. Maybe a happy consequence of the changes to the initializer?

Reply:
Eh, I'm not sure. x-mobi8-ebook is not by default in the list that comes with Marcel so I had to use the extend() method to add the custom type, I can certainly try removing it tho, so I'll modify that this week

Reply:
I meant we don't seem to need the `filetype == "azw3" ? "application/x-mobi8-ebook"` part in step definitions anymore (the tests run fine without it, at least locally). The initializer is good as is, and adding this non-standard format by hand indeed necessary. Sorry for the confusion.

Reply:
Done